doc/book/app_server/luajit_getmetrics.rst

```diff
@@ -0,0 +1,441 @@
+
+LuaJIT getmetrics
+=================
+
+Tarantool can return metrics of a current instance via the Lua API or the C API.
+
+..  contents::
+    :local:
+    :depth: 2
+
+.. _luajit_getmetrics_getmetrics:
+
+misc.getmetrics()
+~~~~~~~~~~~~~~~~~
+
+.. function:: getmetrics()
+
+    Get the metrics values into a table.
+   
+    Parameters: none
+     
+    :return: table
+
+    Example: ``metrics_table = misc.getmetrics()``  
+
+.. _luajit_getmetrics_tablevalues:
+
+getmetrics table values
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The metrics table contains 19 values.
+All values have type = 'number' and are the result of a cast to double, so there may be a very slight precision loss.
+Values whose names begin with ``gc_`` are associated with the
+`LuaJIT garbage collector <http://wiki.luajit.org/New-Garbage-Collector/>`_
+Values whose names begin with ``jit_`` are associated with the
+`"phases" <https://en.wikipedia.org/wiki/Tracing_just-in-time_compilation>`_
+of the just-in-time process; a fuller study of JIT phases can be found at
+`A masters thesis from cern.ch <http://cds.cern.ch/record/2692915/files/CERN-THESIS-2019-152.pdf?version=1>`_.
+
+Values described as "monotonic" are cumulative, that is, they are "totals since
+all operations began", rather than "since the last getmetrics() call".
+Overflow is possible.
+
+Because many values are monotonic,
+a typical analysis involves calling ``getmetrics()``, saving the table,
+calling ``getmetrics()`` again and comparing the table to what was saved.
+The difference is a "slope curve".
+An interesting slope curve is one that shows acceleration,
+for example the difference between the latest value and the previous
+value keeps increasing.
+Some of the table members shown here are used in the examples that come later in this section.
+
+.. container:: table
+
+    .. rst-class:: left-align-column-1
+    .. rst-class:: left-align-column-2
+    .. rst-class:: left-align-column-3
+
+    +----------------------+--------------------------------------------------+------------+
+    | Name                 | Content                                          | Monotonic? |
+    +======================+==================================================+============+
+    | gc_allocated         | number of bytes of allocated memory              | yes        |
+    +----------------------+--------------------------------------------------+------------+
+    | gc_cdatanum          | number of allocated cdata objects                |            |
+    +----------------------+--------------------------------------------------+------------+
+    | gc_freed             | number of bytes of freed memory                  | yes        |
+    +----------------------+--------------------------------------------------+------------+
+    | gc_steps_atomic      | number of steps of garbage collector,            | yes        |
+    |                      | atomic                                           |            |
```

Comment:
Nit: Should we add "phases" at the end of the sentence as it is done for `gc_steps_sweep` description?
Feel free to ignore.

Reply:
**** Okay, I added "phases, incremental" at the end